fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -581,13 +586,14 @@ ACTOR Future<Void> getResolution(CommitBatchContext* self) {
 	// resolution processing but is still using CPU
 	ProxyCommitData* pProxyCommitData = self->pProxyCommitData;
 	std::vector<CommitTransactionRequest>& trs = self->trs;
+	state Span span("MP:getResolution"_loc, self->span.context);
 
 	ResolutionRequestBuilder requests(
 		pProxyCommitData,
 		self->commitVersion,
 		self->prevVersion,
 		pProxyCommitData->version,
-        self->span
+        span
```

Comment:
Was `git clang-format` applied here?

Reply:
Not sure what exactly happened here, probably related to me resolving a merge conflict. Should be fixed now.